substrait_consumer/tests/functional/relations/join_relation_snapshots/IsthmusProducer/left_anti_join_plan.json

```diff
@@ -0,0 +1,232 @@
+{
+  "extensionUris": [{
+    "extensionUriAnchor": 1,
+    "uri": "/functions_boolean.yaml"
+  }, {
+    "extensionUriAnchor": 2,
+    "uri": "/functions_comparison.yaml"
+  }],
+  "extensions": [{
+    "extensionFunction": {
+      "extensionUriReference": 1,
+      "name": "not:bool"
+    }
+  }, {
+    "extensionFunction": {
+      "extensionUriReference": 2,
+      "functionAnchor": 1,
+      "name": "equal:any_any"
+    }
+  }],
+  "relations": [{
```

Comment:
Note that DuckDB optimized out the join so we're not testing a join.

Reply:
I think you mean Isthmus here.  DuckDB wasn't able to produce a plan here due to `INTERNAL Error: Unsupported join type ANTI`